app/views/common_parameters/_parameters_table.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="tab-pane" id="params">
+  <div class="tab-content">
+    <ul class="nav nav-tabs" data-tabs="tabs">
```

Comment:
Perhaps:

1. make global parameters the first tab in the class list under Puppet Parameters (e.g. "Global", "apache", "ntp")
2. have them as separate host level tabs: Host /  Puppet Classes / Puppet Parameters / Global Parameters / Additional Info
3. roll Puppet class parameters into the Puppet Classes tab